product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx

```diff
@@ -10,23 +10,19 @@ A [probe](01_pem_probes/#pem_probes) is a scheduled task that returns a set of p
 
 The `Manage Probes` tab provides a set of `Quick Links` that you can use to create and manage probes:
 
-> -   Click the `` `Manage Custom Probes `` &lt;pem_custom_probes><span class="title-ref"> icon to open the </span><span class="title-ref">Custom Probes</span>\` tab and create or modify a custom probe.
-> -   Click the `` `Copy Probes `` &lt;copy_probe_config><span class="title-ref"> icon to open the </span><span class="title-ref">Copy Probe</span>\` dialog, and copy the probe configurations from the currently selected object to one or more monitored objects.
+-   Click the `` `Manage Custom Probes `` &lt;pem_custom_probes><span class="title-ref"> icon to open the </span><span class="title-ref">Custom Probes</span>\` tab and create or modify a custom probe.
```

Comment:
Please check this on localhost if this is working as expected

Reply:
Hi Abhilasha,
I will be fixing these issues in my next PR, So please approve this.

Reply:
Okay